src/assets/samples/courses.json

```
@@ -430,99 +430,107 @@
+        "title": "Cloud Storage",
         "items": [
-          "Version 17",
-          "React Router",
-          "React Testing Library",
-          "Functional components",
-          "Styled-components",
-          "useState",
-          "useEffect",
-          "Custom Hooks",
-          "Context Api"
+          "S3",
+          "Amazon EBS",
+          "Amazon EFS",
+          "Amazon Glacier",
+          "AWS Storage Gateway"
         ]
       },
       {
-        "title": "Java",
+        "title": "Cloud Computing",
+        "items": ["EC2", "AWS Lambda", "AWS Elastic Beanstalk"]
+      },
+      {
+        "title": "Cloud Database",
         "items": [
-          "Klassen",
-          "Interfaces",
-          "Vererbung",
-          "Objektorientierte Programmierung",
-          "Collections",
-          "Streaming API",
-          "Spring Boot Web",
-          "Spring Data MongoDB",
-          "Spring Security"
+          "Amazon RDS",
+          "Amazon DynamoDB",
+          "Amazon Redshift",
+          "SQL",
+          "NoSQL"
         ]
       },
       {
-        "title": "Datenbanken",
-        "items": ["NoSQL", "MongoDB", "Daten-Normalisierung"]
+        "title": "Agile Workflow",
+        "items": ["Daily Stand-Ups ", "Kanban & Scrum"]
       },
       {
-        "title": "DevOps",
+        "title": "Autoscaling & Fault Tolerance",
         "items": [
-          "Continuous Integration (CI) mit GitHub Actions",
-          "Continuous Deployment (CD) mit GitHub Actions",
-          "Hosting mit Heroku",
-          "AWS Cloud Basics",
-          "Docker"
+          "Elastic Load Balancing",
+          "Reliability / High Availability",
+          "Amazon EC2 Autoscaling",
+          "Amazon CloudFront CDN"
         ]
       },
       {
-        "title": "Testing & Debugging",
+        "title": "Security",
         "items": [
-          "Test-Driven-Development",
-          "Unit-Tests mit JUnit 5",
-          "Mockito",
-          "Hamcrest",
-          "Chrome Dev Tools"
+          "Data Security / Cyber threats ",
+          "Security life cycle ",
+          "Threat mitigation Anti-Malware ",
+          "IAM - Identity management ",
+          "Security compliance ",
+          "AWS Trusted Advisor"
         ]
       },
       {
-        "title": "Weitere Lehrinhalte",
+        "title": "Programming & Tools",
         "items": [
-          "Clean Code",
-          "Arbeiten im Pairing und in Gruppen",
-          "Git-Workflow mit GitHub",
-          "Agile Methoden mit Scrum und Kanban",
-          "UX-Grundkenntnisse",
-          "IntelliJ",
-          "Swagger"
+          "Python Fundamentals ",
+          "Automation ",
+          "Continuous Integration (CI) mit GitHub Actions",
+          "Continuous Deployment (CD) mit GitHub Actions",
+          "JSON / YAML  ",
+          "VSCode  ",
+          "API und REST ",
+          "Git Workflow ",
+          "Markdown ",
+          "Pair Programming"
```

Don't know if that was on purpose or not 🤷‍♀️ 
```suggestion
          "Python Fundamentals",
          "Automation",
          "Continuous Integration (CI) mit GitHub Actions",
          "Continuous Deployment (CD) mit GitHub Actions",
          "JSON / YAML",
          "VSCode",
          "API und REST",
          "Git Workflow",
          "Markdown",
          "Pair Programming"
```

---

thanks :)